packages/mobile/locales/en-US/nuxNamePin1.json

```diff
@@ -10,8 +10,8 @@
     "heading1": "Data and Privacy",
     "heading2": "Celo Assets and {{appName}} Account",
     "info": "In order to use our services, please read and accept our <0>User Agreement and Terms</0> by clicking the accept button below.",
-    "privacy": "By joining this network, you give us permission to collect anonymous information about your use of the app. Additionally, if you confirm your phone number, a hashed copy of it will be stored on the Celo Network. If you grant Valora access to your contact list, Valora will import each contact's name, phone number and profile picture as described in our <0>Privacy Policy</0>.",
-    "goldDisclaimer": "When you create an ‘account’ with {{appName}} you are creating a digital wallet to which only you hold the keys. No other person or entity, including cLabs or Valora, can recover your key, change or undo transactions, or recover lost funds. Be aware that CELO, cUSD, and cEUR are part of a new asset class and present a risk of financial loss.  Carefully consider your financial circumstances and tolerance for financial risk before purchasing any asset. You can only send up to {{dailyLimitCUSD}} cUSD per day through {{appName}} (the “Daily Limit”).  You must contact support@cLabs.co and provide additional, personal information to request increasing this Daily Limit. There could be a delay to conduct required compliance."
+    "privacy": "By joining this network, you give us permission to collect anonymous information about your use of the app. Additionally, if you confirm your phone number, a hashed copy of it will be stored on the Celo Network. If you grant Valora access to your contact list, Valora will import contacts' name, phone number and profile picture to allow users to connect through the Valora app. To learn how we collect and use this information please review our <0>Privacy Policy</0>.",
```

Comment:
i'm still confused about the grammar here.  specifically `Valora will import contacts' name`.  we're missing an article, or the plurality is wrong, or I'm just confused.  the previous version seemed clear to me:  `Valora will import each contact's name`

Reply:
Done